Sources/UIKit/AdaptableTextContainer.swift

```diff
@@ -147,19 +147,19 @@ extension UISegmentedControl {
     // `UISegmentedControl` has terrible generics ([NSObject: AnyObject]? or [AnyHashable: Any]?) on
```

Comment:
Does this comment need to be updated at all in light of these changes?

Reply:
I fixed an error inside that method. When I changed the return type back to being non-optional I forgot to default to an empty dictionary when nil. I also added a test for segmented control similar to the other UIKit tests.

As for the comment, I can't speak to whether it is still appropriate or not, although I can say in Swift 5 `titleTextAttributes(for:)` returns `[NSAttributedFont.Key: Any]?`.